clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -6612,6 +6612,8 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
   Args.AddLastArg(CmdArgs, options::OPT_fconstexpr_depth_EQ);
   Args.AddLastArg(CmdArgs, options::OPT_fconstexpr_steps_EQ);
 
+  Args.AddLastArg(CmdArgs, options::OPT_fimplicit_constexpr);
```

Comment:
We should issue a driver diagnostic *somewhere* if someone tries to pass `-fimplicit-constexpr` if we’re not compiling C++. And we’d need tests for that too then.

Reply:
I have added check for c++, but it would be goot to enable for 14+ only (but not sure how to do it in this file, will look at it tomorrow)